DataFormats/Detectors/TRD/include/DataFormatsTRD/Constants.h

```diff
@@ -22,74 +22,75 @@ namespace trd
-constexpr int MAXMCMCOUNT = 69120;             // at most mcm count maxchamber x nrobc1 nmcmrob
-constexpr int MAXLINKERRORHISTOGRAMS = 10;     // size of the array holding the link error plots from the raw reader
-constexpr int MAXPARSEERRORHISTOGRAMS = 60;    // size of the array holding the parsing error plots from the raw reader
-constexpr unsigned int ETYPEPHYSICSTRIGGER = 0x2;     // CRU Half Chamber header eventtype definition
-constexpr unsigned int ETYPECALIBRATIONTRIGGER = 0x3; // CRU Half Chamber header eventtype definition
-constexpr int MAXCRUERRORVALUE = 0x2;          // Max possible value for a CRU Halfchamber link error. As of may 2022, can only be 0x0, 0x1, and 0x2, at least that is all so far(may2022).
+constexpr int HBFBUFFERMAX = 1048576;                 //< max buffer size for data read from a half cru, (all events)
+constexpr unsigned int CRUPADDING32 = 0xeeeeeeee;     //< padding word used in the cru.
+constexpr int CHANNELNRNOTRKLT = 23;                  //< this marks channels in the ADC mask which don't contribute to a tracklet
+constexpr int NOTRACKLETFIT = 31;                     //< this value is assigned to the fit pointer in case no tracklet is available
+constexpr int TRACKLETENDMARKER = 0x10001000;         //< marker for the end of tracklets in raw data, 2 of these.
+constexpr int PADDINGWORD = 0xeeeeeeee;               //< half-CRU links will be padded with this words to get an even number of 256bit words
+constexpr int DIGITENDMARKER = 0x0;                   //< marker for the end of digits in raw data, 2 of these
+constexpr int MAXDATAPERLINK32 = 13824;               //< max number of 32 bit words per link ((21x12+2+4)*64) 64 mcm, 21 channels, 10 words per channel 2 header words(DigitMCMHeader DigitMCMADCmask) 4 words for tracklets.
+constexpr int MAXDATAPERLINK256 = 1728;               //< max number of linkwords per cru link. (256bit words)
+constexpr int MAXEVENTCOUNTERSEPERATION = 200;        //< how far apart can subsequent mcmheader event counters be before we flag for concern, used as a sanity check in rawreader.
+constexpr int MAXMCMCOUNT = 69120;                    //< at most mcm count maxchamber x nrobc1 nmcmrob
+constexpr int MAXLINKERRORHISTOGRAMS = 10;            //< size of the array holding the link error plots from the raw reader
+constexpr int MAXPARSEERRORHISTOGRAMS = 60;           //< size of the array holding the parsing error plots from the raw reader
+constexpr unsigned int ETYPEPHYSICSTRIGGER = 0x2;     //< CRU Half Chamber header eventtype definition
+constexpr unsigned int ETYPECALIBRATIONTRIGGER = 0x3; //< CRU Half Chamber header eventtype definition
+constexpr int MAXCRUERRORVALUE = 0x2;                 //< Max possible value for a CRU Halfchamber link error. As of may 2022, can only be 0x0, 0x1, and 0x2, at least that is all so far(may2022).
```

Comment:
Does this actually make a difference? Is a doxygen doc generated for this namespace? In any case if we change this please use `///<` as in the commenting guidelines: https://rawgit.com/AliceO2Group/CodingGuidelines/master/comments_guidelines.html?showone=Class_Data_Members#Class_Data_Members